app/resources/views/pages/location_search/spreadsheet.blade.php

```diff
@@ -1,4 +1,18 @@
 	<div class="spreadsheet sort-by-{{ $order_by }}">
+	<div class="spreadsheet form-container">
+			<form class="form-inline">
```

Comment:
Indentation is messed up a bit here.  The <form doesn't line up with the closing /form.

There isn't much point in using a form here.  A form would be useful if we planned to submit the form and get a resulting page that way but we're using AJAX instead.  JavaScript is doing everything.  It seems better to bind to the click event of the "Update" button rather than binding to the submit of a form that would best be removed.